paasta_tools/cli/cmds/status.py

```diff
@@ -1516,17 +1524,21 @@ def get_replica_state(pod: KubernetesPodV2) -> ReplicaState:
     phase = pod.phase
     state = ReplicaState.UNKNOWN
     reason = pod.reason
-    if phase == "Failed" or reason == "Evicted":
-        state = ReplicaState.NOT_RUNNING
+    if reason == "Evicted":
+        state = ReplicaState.EVICTED
+    elif phase == "Failed":
+        state = ReplicaState.FAILED
     elif phase is None or not pod.scheduled:
         state = ReplicaState.UNSCHEDULED
     elif pod.delete_timestamp:
         state = ReplicaState.TERMINATING
     elif phase == "Pending":
-        if not pod.containers or all([c.state == "waiting" for c in pod.containers]):
-            state = ReplicaState.STARTING
+        if not pod.containers:
+            state = ReplicaState.NO_CONTAINERS_YET
+        elif all([c.state == "waiting" for c in pod.containers]):
```

Comment:
just double-checking: the state is lowercase in the k8s api response?

Reply:
Interesting, [the docs](https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#container-states) indicate it should be `Waiting`.

Reply:
@EvanKrall perhaps should we `c.state.lower()` just to be safe here?

or try to catch a pod in this state and see if the original code was right? :p